src/metax_api/services/file_service.py

```diff
@@ -84,6 +84,10 @@ def get_queryset_search_params(cls, request):
                 cls.check_user_belongs_to_project(request, project)
             queryset_search_params['project_identifier'] = project
 
+            if request.query_params.get('file_path', False):
+                path = request.query_params['file_path']
+                queryset_search_params['file_path__contains'] = path
```

Comment:
I dont think the ``path`` middle-step variable brings much value here.

Original spec was lacking in detail, but this should ensure that the query param project_identifier is also specified when using this query parameter. I.e., just check if project_identifier is missing, and raise an error if so.